static/src/javascripts/projects/common/modules/experiments/tests/contributions-moment-banner-environment.js

```diff
@@ -0,0 +1,111 @@
+// @flow
+
+// import { getSync } from 'lib/geolocation';
+import { acquisitionsBannerMomentTemplate } from 'common/modules/commercial/templates/acquisitions-banner-moment.js';
+import { canShowBannerSync } from 'common/modules/commercial/contributions-utilities';
+import { getSync as geolocationGetSync } from 'lib/geolocation';
+
+const geolocation = geolocationGetSync();
+const campaignId = 'environment_campaign';
+
+const defaultLeadSentence =
+    'The climate emergency is the defining issue of our times. ';
+const defaultCopy =
+    'This is The Guardian’s pledge: we will be truthful, resolute and undeterred in pursuing our journalism on the environment. We will give global heating, wildlife extinction and pollution the urgent attention they demand. Our independence means we can interrogate inaction by those in power. It means our reporting will always be grounded in scientific facts, never in commercial or political interests. Support from our readers makes this work possible.';
+const defaultMobileCopy =
+    'This is The Guardian’s pledge: we will be truthful, resolute and undeterred in pursuing our journalism on the environment. Support from our readers makes this work possible.';
+
+const thankYouCopy =
+    'The climate emergency is the defining issue of our times. This is The Guardian’s pledge: we will be truthful, resolute and undeterred in pursuing our journalism on the environment. We will give global heating, wildlife extinction and pollution the urgent attention they demand. Our independence means we can interrogate inaction by those in power. ';
+const thankYouMobileCopy =
+    'The climate emergency is the defining issue of our times. This is The Guardian’s pledge: we will be truthful, resolute and undeterred in pursuing our journalism on the environment. ';
```

Comment:
Probably not important - but there's a stray space after the full stop on this one.